### PR TITLE
Slack Dump

### PR DESCRIPTION
**Pull request by ManishaBayya**
Please review the code

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,31 +0,0 @@
-from setuptools import setup, find_packages
```
Comment (shivam05011996) [inline setup.py:1]: When we import modules, we should first write system modules, and then other modules, in this case, import os should appear first.

File: setup.py [hunk 2]
```diff
@@ -1,31 +0,0 @@
...
-
```
Comment (shivam05011996) [inline setup.py:2]: Please remove this empty line.

File: slack_dump/slack_dump.py [hunk 3]
```diff
@@ -1,56 +0,0 @@
...
-            start1 = r['ts']
-    get_real_time_message(token, start1)
-
-def get_real_time_message(token, start1):
-    try:
-        slack = SlackClient(token)
-        while not slack.rtm_connect():
-            time.sleep(CONNECTION_SLEEP_INTERVAL)
-        end = format(time.time(), '0.6f')
-
```
Comment (shivam05011996) [inline slack_dump/slack_dump.py]: We dont require empty line here

File: slack_dump/slack_dump.py [hunk 4]
```diff
@@ -1,56 +0,0 @@
...
-                continue
-	    print (msg)
-
-            if not len(msg[0]) == 1:
-                try:
-                    key = hashlib.sha1(msg[0]['channel'] + msg[0]['ts']).hexdigest()
-		    msg[0]["key"] = key
-		    res=db.slack_db.insert_one(msg[0])
-		    print ("Inserted!!")
-
```
Comment (shivam05011996) [inline slack_dump/slack_dump.py]: We don't need empty line here.

File: LICENSE [hunk 5]
```diff
@@ -1,21 +0,0 @@
-MIT License
```
Comment (shivam05011996) [inline LICENSE]: Have we actually taken MIT License? Please confirm from @prashanthellina , regarding the use of this.

**Review by shivam05011996 — CHANGES_REQUESTED** [review]
Please always use branch while sending pull request.
Rest of the changes to made are added.

File: LICENSE (hunk 5 above)
Comment (prashanthellina) [reply LICENSE]: @shivam-deepcompute MIT is totally fine.

**Comment by prashanthellina** [comment]
@supriyopaul Can you please review and post your feedback? Thank you.

**Comment by supriyopaul** [comment]
Please update the README.md  with how to install and run the program

**Pull request closed without merging**